ibis-server/resources/function_list/clickhouse.csv

```diff
@@ -1,110 +1,30 @@
-scalar,upper,String,"Converts string to uppercase."
-scalar,trim,String,"Removes leading and trailing whitespace."
-scalar,ltrim,String,"Removes leading whitespace."
-scalar,rtrim,String,"Removes trailing whitespace."
-scalar,reverse,String,"Reverses string."
-scalar,replaceAll,String,"Replaces all occurrences of substring."
-scalar,replaceRegexpAll,String,"Replaces all regex matches."
-scalar,empty,UInt8,"Checks if array is empty."
-scalar,notEmpty,UInt8,"Checks if array is not empty."
-scalar,arrayLength,UInt64,"Returns length of array."
-scalar,arrayConcat,Array,"Concatenates arrays."
-scalar,arrayElement,Any,"Returns array element at specified index."
-scalar,has,UInt8,"Checks if array contains value."
-scalar,indexOf,UInt64,"Returns index of first occurrence of element."
-scalar,toYear,UInt16,"Extracts year from date/datetime."
-scalar,toQuarter,UInt8,"Extracts quarter from date/datetime."
-scalar,toMonth,UInt8,"Extracts month from date/datetime."
-scalar,toDayOfMonth,UInt8,"Extracts day of month from date/datetime."
-scalar,toDayOfWeek,UInt8,"Extracts day of week from date/datetime."
-scalar,toHour,UInt8,"Extracts hour from datetime."
-scalar,toMinute,UInt8,"Extracts minute from datetime."
-scalar,toSecond,UInt8,"Extracts second from datetime."
-scalar,now,DateTime,"Returns current date and time."
-scalar,today,Date,"Returns current date."
-scalar,yesterday,Date,"Returns yesterday's date."
-scalar,formatDateTime,String,"Formats date/time according to given format."
-scalar,JSONHas,UInt8,"Checks if JSON field exists."
-scalar,JSONLength,UInt64,"Returns length of JSON array or object."
-scalar,JSONType,String,"Returns type of JSON value."
-scalar,JSONExtractString,String,"Extracts string from JSON."
-scalar,JSONExtractFloat,Float64,"Extracts float from JSON."
-scalar,JSONExtractInt,Int64,"Extracts integer from JSON."
-scalar,IPv4NumToString,String,"Converts IPv4 to string."
-scalar,IPv4StringToNum,UInt32,"Converts string to IPv4."
-scalar,IPv6NumToString,String,"Converts IPv6 to string."
-scalar,IPv6StringToNum,FixedString(16),"Converts string to IPv6."
-window,rowNumberInAllBlocks,UInt64,"Returns row number across all blocks."
-window,rowNumberInBlock,UInt64,"Returns row number within current block."
-window,rank,UInt64,"Returns rank of current row with gaps."
-window,denseRank,UInt64,"Returns rank of current row without gaps."
-window,runningDifference,Same as input,"Returns difference between successive values."
-window,lagInFrame,Same as input,"Returns value from previous row in frame."
-window,leadInFrame,Same as input,"Returns value from next row in frame."
-window,firstValue,Same as input,"Returns first value in window frame."
-window,lastValue,Same as input,"Returns last value in window frame."
-window,nthValue,Same as input,"Returns nth value in window frame."
+function_type,name,return_type,param_names,param_types,description
+aggregate,min,Same as input,,Any,"Returns the minimum value."
```

Comment:
```suggestion
aggregate,min,Same as input,,Any,"Returns the minimum value."
```
The return type is weird 🤔. I think they're duplicated with the datafusion default functions. Maybe we can just remove them.